package.json

```
@@ -113,7 +113,7 @@
     "@babel/parser": "7.25.6",
     "@babel/traverse": "7.25.6",
     "@babel/types": "7.25.6",
-    "@hono/vite-dev-server": "0.18.0",
+    "@hono/vite-dev-server": "^0.18.0",
```

Maybe we should require the new fix?

```suggestion
    "@hono/vite-dev-server": "^0.18.1",
```